### ISSUE TITLE
Context menu options missing from Pocket widget

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#9140]
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
Context menu options missing from Pocket widget

- Platform (Win7, 8, 10? macOS? Linux distro?):
All

- Brave Version (revision SHA):
Brave	0.16.0
rev	be76cd3
Muon	3.2.101

- Steps to reproduce:
    1. Enable Pocket from about:preferences#extensions
    2. Login to pocket 
    3. Right click doesn't bring up the context menu. Only way to access options page is to login to pocket in a new tab and select from profile
    
- Actual result:
Context menu options missing from Pocket widget

- Expected result:
Should show context menu similar to Chrome(screenshot)

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
Yes

- Can this issue be consistently reproduced?
Yes

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:
![image](https://cloud.githubusercontent.com/assets/17010094/26589190/4b4762b0-454f-11e7-956d-374765d63548.png)

- Any related issues:
cc: @kevinlawler @jonathansampson 
Setting as 1.0 please change milestone if required

**Comment by PostPollux** [comment]
Yes, please! Add that context menu. It's super easy to save a page to pocket, but accessing your pocket collection is very annoying right now... Would make using pocket much more fun!
